.travis.yml

```diff
@@ -16,6 +19,8 @@ geosx_osx_build: &geosx_osx_build
   os: osx
   osx_image: xcode13.2
   install:
+  - echo "running nproc"; nproc
+  - top -l 1 -s 0 | grep PhysMem
```

Comment:
Do we need those lines merged?

Reply:
I'm fine, either way. We can keep it for debugging purposes or remove for cleanliness.

Reply:
I'll think we'll find the cleaner way to get this information after all our final moves.

You're next in line in the merge queue for both incoming PRs.
Can you merge? Consider the order you think most appropriate for merging 😉

Reply:
> Can you merge? Consider the order...

I do not have merge authority.
Feel free to merge this PR first.